### DISCUSSION TITLE
Sound triggered by Statuseffect/Event never stops playing in Multiplayer

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14009] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Sound triggered by Statuseffect/Event never stops playing in Multiplayer and loops forever

Example:
The sound plays for 3 seconds in SP just fine, but keeps playing in MP

Part of larger series of statuseffect MP desyincs  #14008

```xml
    <ScriptedEvent identifier="loopingsoundmpbug">
          <TagAction criteria="player" tag="player" />
          <StatusEffectAction targettag="player">
              <StatusEffect target="this" duration="3" >
                <Sound file="Content/Sounds/Radiostatic.ogg" range="4000" volume="2" loop="true" />
              </StatusEffect>
            </StatusEffectAction>            
    </ScriptedEvent >
```

### Reproduction steps

Use `triggerevent loopingsoundmpbug` in both SP and MP with the above event

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of #13406

**Comment by Regalis11** [comment]
Nvm, closed the wrong discussion, https://github.com/FakeFishGames/Barotrauma/discussions/14008 is the duplicate one :D

**Comment by Jasontti** [comment]
Tested the fix for this and no issues found.
Closing discussion as resolved.
